csharp/Program.cs

```diff
@@ -6,7 +6,55 @@ class Program
     {
         static void Main(string[] args)
         {
-            Console.WriteLine("Hello C#!");
+            Console.WriteLine("Ingrese numero de tarjeta de credito");
+            string numeroTarjeta = Console.ReadLine();
+            Console.WriteLine(getIssuer(numeroTarjeta));
+
         }
+
+        public static string getIssuer(string tarjeta)
+        {
+            string op1 = tarjeta[0].ToString();
+            string op2 = tarjeta[1].ToString();
```

Comment:
Please identify magic numbers in all your code and lets define constants for those. Constants names should reflect what is their purpose